Helper/Reminder.php

```diff
@@ -61,10 +61,13 @@ protected function purchaseData(\Magento\Sales\Model\Order $order)
         $name   = $this->purchaseHelper->customerName($order);
         $lang   = $this->localeHelper->getStoreLocale();
         $date   = $this->purchaseHelper->createdAt($order);
+        $customerId = $order->getCustomerId() ? $order->getCustomerId() : '';
```

Comment:
@AdamLipscore what do you think about aligning assignments and hashes by `=` or `=>`
like 
```
$lang       = $this->localeHelper->getStoreLocale();
$date       = $this->purchaseHelper->createdAt($order);
$customerId = $order->getCustomerId() ? $order->getCustomerId() : '';
```
we use this approach in the extension code, so if you ok with it, then let's keep it and fix this assignment block and hash below.
If not, it's also fine, please let me know so I won't comment it in the PRs in the future.